src/core/lib/support/block_annotate.h

```diff
@@ -26,8 +26,12 @@
 #define GRPC_SCHEDULING_START_BLOCKING_REGION \
   do {                                        \
   } while (0)
-#define GRPC_SCHEDULING_END_BLOCKING_REGION \
-  do {                                      \
+#define GRPC_SCHEDULING_END_BLOCKING_REGION_NO_EXEC_CTX \
```

Comment:
It seems a bit messy to put code that references exec_ctx into gpr.  I realize that this is just a macro, so there's no link-time dependency, but it still seems sub-optimal.  Do we have any reasonable alternative?